src/src/main/avro/event_issuance/issuance.avsc

```diff
@@ -164,6 +164,67 @@
             {
               "name": "enforcement_org_unit_cd_txt",
               "type": "string"
+            },
+            {
```

Comment:
@amlanc1 , ETK is still sending the counts field in the payload. I'll be sending to the violations directly using DB Adapter.